cmd/puppeth/ssh.go

```diff
@@ -77,7 +77,23 @@ func dial(server string, pubkey []byte) (*sshClient, error) {
 		log.Warn("No SSH key, falling back to passwords", "path", path, "err", err)
 	} else {
 		key, err := ssh.ParsePrivateKey(buf)
-		if err != nil {
+		if string(err.Error()) == "ssh: cannot decode encrypted private keys" {
+
```

Comment:
Please no empty lines here :)

Reply:
So in general, no empty lines padding the beginning or end of blocks?